library/src/test/java/com/novoda/downloadmanager/LiteBatchFileBuilderTest.java

```diff
@@ -0,0 +1,99 @@
+package com.novoda.downloadmanager;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class LiteBatchFileBuilderTest {
+
+    private static final String ANY_NETWORK_ADDRESS = "http://ak.assets.com/some-file";
+
+    private final InternalBatchBuilder batchBuilder = mock(InternalBatchBuilder.class);
+    private final StorageRoot storageRoot = mock(StorageRoot.class);
+    private final DownloadBatchId downloadBatchId = mock(DownloadBatchId.class);
+
+    private LiteBatchFileBuilder liteBatchFileBuilder;
+
+    @Before
+    public void setUp() {
+        when(downloadBatchId.rawId()).thenReturn("my-movie");
```

Comment:
Can use `BDD` mockito. `given().willReturn`.

Reply:
Done in d5a16cbdfb291c625e4d91fccd0c0d3d7bc269fc.